### PR TITLE
Add storage functionality

### PR DESCRIPTION
**Pull request by sedroche**
The new fh-dataman service needs to get a connection to the primary node in the replica set. This PR exposes this functionality from this [PR](https://github.com/feedhenry/fh-mbaas/pull/31) in fh-mbaas

### CODE REVIEW

**Comment by sedroche** [comment]
Retest this please

**Comment by maleck13** [comment]
@sedroche this looks reasonable but I would defer to @nialldonnellyfh on it really

File: lib/mbaasRequest/mbaasRequest.js [hunk 1]
```diff
@@ -317,6 +317,30 @@ function _buildAppRequestParams(params) {
   };
 }
 
+/**
+ * Build request params for a storage request to an Mbaas
+ * @param params
+ * @returns Object
+ * @private
+ */
+function _buildStorageRequestParams(params) {
```
Comment (fh-qube) [inline lib/mbaasRequest/mbaasRequest.js]: INFO Rename this '_buildStorageRequestParams' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Pull request closed without merging**